.github/workflows/prebuild.yml

```
@@ -28,6 +28,18 @@ jobs:
           platform: android
           arch: arm
           flags: -D ANDROID_STL=c++_shared
+        - os: ubuntu-22.04
+          platform: linux
+          arch: x64
+          tags: -musl
+          flags: --environment musl
+          musl: x86_64-linux-musl-cross-x86_64-linux
+        - os: ubuntu-22.04
+          platform: linux
+          arch: arm64
+          tags: -musl
+          flags: --environment musl
+          musl: x86_64-linux-musl-cross-aarch64-linux
```

```suggestion
          musl: aarch64-linux-musl-cross-aarch64-linux
```

The workflow will produce two prebuilds for `x86_64-linux-musl` instead of one per architecture.